src/components/Chip.tsx

```diff
@@ -0,0 +1,65 @@
+import { Ref, forwardRef } from 'react'
+import { Flex, FlexProps, P, Spinner } from 'honorable'
+import PropTypes from 'prop-types'
+
+type ChipProps = FlexProps & {
+  size?: 'medium' | 'large' | string
+  severity?: 'neutral' | 'info' | 'success' | 'warning' | 'error' | string
+  loading?: boolean
+}
+
+const propTypes = {
+  size: PropTypes.oneOf(['medium', 'large']),
+  severity: PropTypes.oneOf(['neutral', 'info', 'success', 'warning', 'error']),
+  loading: PropTypes.bool,
+}
+
+const severityToColor = {
+  neutral: 'text-light',
+  info: 'text-primary',
+  success: 'text-success',
+  warning: 'text-warning',
+  error: 'text-error',
+}
+
+function ChipRef({
+  children,
+  size = 'medium',
+  severity = 'neutral',
+  loading = false,
```

Comment:
I think we should have a generic icon option as well, it looks like there are a few designs where chips need these